TCAT/AppDelegate.swift

```diff
@@ -30,6 +31,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         
+        // Set Up Google Services
+        FirebaseApp.configure()
+        GMSServices.provideAPIKey(Keys.googleMaps.value)
+        GMSPlacesClient.provideAPIKey(Keys.googlePlaces.value)
+        
+        migrationToNewPlacesModel()
```

Comment:
I think adding a check here for some migration key would be ideal, rather than in the function itself. I would be cautious about using the version number because someone could feasibly jump multiple versions. Storing version number of the last migration may suffice because then one could check if the last migration was up to date, in case we ever need to migrate again. For example, v1.0.1 we add first migration from v1.0.0, then in v1.0.3 we need an even newer migration, check if version is at least something but less than latest migration. I'm sure you can think of specifics.